index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CALC</title>
+  </head>
+
+  <body>
+    <div>
+      <input class="inputsize" type="text" id="Input" />
+    </div>
+      <button id="Divide">/</button>
+      <button id="Multiply">*</button>
+      <button id="Subtract">-</button>
+      <button id="Addition">+</button>
+    </div>
```

Comment:
this needs an opening div above line 15, right now you just have a lone closing </div>